encoding/thrift/thriftrw-plugin-yarpc/main.go

```diff
@@ -243,16 +263,33 @@ func (generator) Generate(req *api.GenerateServiceRequest) (*api.GenerateService
 			parentModule = req.Modules[parent.ModuleID]
 		}
 
+		unary := *_unaryHandlerWrapper
+		unaryImport := unary[0:strings.LastIndex(unary, ".")]
+		unaryWrapper := unary[strings.LastIndex(unary, ".")+1 : len(unary)]
+
+		oneway := *_onewayHandlerWrapper
+		onewayImport := oneway[0:strings.LastIndex(oneway, ".")]
+		onewayWrapper := oneway[strings.LastIndex(oneway, ".")+1 : len(oneway)]
```

Comment:
pull this into a function (`splitFunctionPath`?) and use a local variable for the calls to `strings.LastIndex`.